src/js/api/config.js

```diff
@@ -61,9 +61,50 @@ function _adjustDefaultBwEstimate(estimate) {
     return Defaults.bandwidthEstimate;
 }
 
+function _copyToLocalization(allOptions) {
+    const { advertising, related, sharing } = allOptions;
+    const localization = Object.assign({}, Defaults.localization, allOptions.localization);
+
+    if (advertising) {
+        localization.advertising = Object.assign({}, localization.advertising, {
```

Comment:
since these map to exactly the same fields we can just do 
`localization.advertising = Object.assign({}, localization.advertising, advertising);`